packages/amplify-codegen-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -97,7 +92,7 @@ type AWSAccountInfo = {
 };
 
 const BUCKET_TEST_REGEX = /test/;
-const IAM_TEST_REGEX = /!RotateE2eAwsToken-e2eTestContextRole|-integtest$|^amplify-|^eu-|^us-|^ap-/;
+const IAM_TEST_REGEX = /!RotateE2eAwsToken-e2eTestContextRole|-integtest|^amplify-|^eu-|^us-|^ap-/;
```

Comment:
I think this regex is wrong. I'm not aware of any functionality of `!` on its own. I think this would be matching the `!` character literally.